### PR TITLE
PR to check my changed

### PR DESCRIPTION
**Pull request by EdwardKaravakis**
PR to check my changed before I notify Yee Li

### CODE REVIEW

**Comment by EdwardKaravakis** [comment]
@wguanicedew please have a quick look before I notify Yee Li - I have added a prod env for panda server and I've corrected some mistakes I've discovered along the way

File: overlays/panda/prod/cnpg-panda-server-database.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  primaryUpdateStrategy: unsupervised
+
+  bootstrap:
+    initdb:
+      database: panda
+      owner: panda
+      secret:
+        name: usdf-panda-server-creds
+  
+  imageName:  ghcr.io/cloudnative-pg/postgresql:14.4-7
```
Comment (wguanicedew) [inline overlays/panda/prod/cnpg-panda-server-database.yaml:18]: For PanDA database, I am not sure this image will work.
PanDA database has some special configuration https://github.com/PanDAWMS/panda-database/blob/main/etc/postgresql.conf#L3-L5.

**Comment by EdwardKaravakis** [comment]
Yes, it was only for you to have a clear diff between the changes

File: overlays/panda/prod/cnpg-panda-server-database.yaml (hunk 1 above)
Comment (EdwardKaravakis) [reply overlays/panda/prod/cnpg-panda-server-database.yaml:18]: hm not sure how to do that / add that, is there any example that I can look at from rucio or butler or anywhere?

Comment (wguanicedew) [reply overlays/panda/prod/cnpg-panda-server-database.yaml:18]: Here there is an example https://github.com/slaclab/rubin-usdf-butler-deploy/blob/main/Dockerfile.

Comment (EdwardKaravakis) [reply overlays/panda/prod/cnpg-panda-server-database.yaml:18]: Hm I am not sure how to add pg_cron and pg_partman_bgw ... how do you add shared preload libraries ? any example?

Comment (wguanicedew) [reply overlays/panda/prod/cnpg-panda-server-database.yaml:18]: https://github.com/PanDAWMS/panda-database/blob/main/Dockerfile#L13-L15.
Here are examples. Tadashi developed it.

Comment (EdwardKaravakis) [reply overlays/panda/prod/cnpg-panda-server-database.yaml:18]: ah wonderful, thanks!

**Pull request closed without merging**